spot-client/src/spot-tv/native-functions/native-controller/native-controller.js

```diff
@@ -0,0 +1,45 @@
+import { logger } from 'common/logger';
+
+/**
+ * Implements a controller that utilizes the event emitter of the native (Electron) process if available.
+ *
+ * Using it will enable SpotTVs running wrapped in Electron access OS-native functions.
+ */
+class NativeControntroller {
+    /**
+     * Some.
+     */
+    constructor() {
+        // eslint-disable-next-line
+        this.ipcRenderer = this.isAvailable() && __non_webpack_require__('electron').ipcRenderer;
```

Comment:
Did you still want to change this to window.require?

Reply:
Right, thanks!